### DISCUSSION TITLE
How can the QML import path be configured

### DISCUSSION THREAD

**Discussion by machinekoder** [seanwu1105/vscode-qt-for-python#311] [Q&A]
How can we configure the QML import path `qmllint` and also the code completion is looking for?

I've tried creating a `.qmllint.ini` file in the project root, but it is unfortunately not picked up.

**Comment by seanwu1105** [comment]
You may try to set the qmllint args in the extension configuration for the functionality you look for. I can provide some working examples but I currently do not have the bandwidth to do it.
